plugins/liip/repaircafe/components/Button.php

```diff
@@ -0,0 +1,135 @@
+                'description'       => 'liip.repaircafe::lang.component.button.label.description',
+                'required'          => true,
+                'type'              => 'string',
+            ],
+            'url' => [
+                'title'             => 'liip.repaircafe::lang.component.button.url.title',
+                'description'       => 'liip.repaircafe::lang.component.button.url.description',
+                'required'          => true,
+                'type'              => 'string',
+            ],
+            'target' => [
+                'title'             => 'liip.repaircafe::lang.component.button.target.title',
+                'description'       => 'liip.repaircafe::lang.component.button.target.description',
+                'default'           => '',
+                'type'              => 'dropdown',
+                'options'           => [
+                    '' => 'liip.repaircafe::lang.component.button.target.option.self',
+                    '_blank' => 'liip.repaircafe::lang.component.button.target.option.blank',
+                ],
+            ],
+            'style' => [
+                'title'             => 'liip.repaircafe::lang.component.button.style.title',
+                'description'       => 'liip.repaircafe::lang.component.button.style.description',
+                'default'           => 'btn-primary',
+                'required'          => true,
+                'type'              => 'dropdown',
+                'options'           => [
+                    'btn-primary' => 'liip.repaircafe::lang.component.button.style.option.btn_primary',
+                    'btn-secondary' => 'liip.repaircafe::lang.component.button.style.option.btn_secondary',
+                    'btn-outline-primary' => 'liip.repaircafe::lang.component.button.style.option.btn_outline_primary',
+                    'btn-outline-secondary' => 'liip.repaircafe::lang.component.button.style.option.btn_outline_secondary',
+                ],
+            ],
+            'position' => [
+                'title'             => 'liip.repaircafe::lang.component.button.position.title',
+                'description'       => 'liip.repaircafe::lang.component.button.position.description',
+                'default'           => '',
+                'type'              => 'dropdown',
+                'options'           => [
+                    '' => 'liip.repaircafe::lang.component.button.position.option.left',
+                    'text-center' => 'liip.repaircafe::lang.component.button.position.option.center',
+                    'text-right' => 'liip.repaircafe::lang.component.button.position.option.right',
+                ],
+            ],
+            'size' => [
+                'title'             => 'liip.repaircafe::lang.component.button.size.title',
+                'description'       => 'liip.repaircafe::lang.component.button.size.description',
+                'default'           => '',
```

Comment:
Also here, not sure, if setting the default makes sense. But if it displays just fine without one, then leave it like it is.

Reply:
same here